XML/TestCases/PerformanceTests.xml

```diff
@@ -854,6 +854,28 @@
       <SetupScript>.\Testscripts\Windows\SETUP-Check-PowerPlan.ps1,.\Testscripts\Windows\SETUP-PERF-Add-NIC.ps1</SetupScript>
     </SetupConfig>
   </test>
+  <test>
+    <testName>PERF-SCHEDULER</testName>
+    <testScript>PERF-SCHEDULER.ps1</testScript>
+    <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\perf_scheduler.sh</files>
+    <TestParameters>
+      <param>MSG_THREADS=SCHBENCH_MSG_THREADS</param>
+      <param>RUNTIME=SCHBENCH_RUNTIME</param>
+      <param>SLEEPTIME=SCHBENCH_SLEEPTIME</param>
+      <param>CPUTIME=SCHBENCH_CPUTIME</param>
+      <param>DATASIZE=HACKBENCH_DATASIZE</param>
+      <param>LOOPS=HACKBENCH_LOOPS</param>
```

Comment:
Are all of these parameters changed frequently?
If not we can move them inside shell script.

Reply:
Not frequently. But these parameters can be defined by users, so I put them here.